django_loose_fk/utils.py

```diff
@@ -7,6 +9,21 @@
 from rest_framework.request import Request
 
 
+def is_local(host: str, url: str) -> bool:
```

Comment:
Could we have a few unittests for only this method? I'm all in favor of integration tests but this feels complicated enough to test separately as well

Reply:
Tests are added